### ISSUE TITLE
GSVAparam error

### ISSUE THREAD

**Issue by akettring** [rcastelo/GSVA#123]
I'm having some issues getting off the ground with this package. Following the vignette, I get to this point...

gsvaPar <- gsvaParam(X, gs)

...but the class doesn't seem to work...

Error in gsvaParam(X, gs) : could not find function "gsvaParam"

Am I missing something obvious here?

Sorry if this is a basic problem. I can't find any information on this. Thanks.

**Comment by axelklenk** [comment]
Hmmm, this function is only available in the most recent version (GSVA 1.50, Bioconductor 3.18, or later).  
Did you install and load GSVA using
```
BiocManager::install("GSVA")
library("GSVA")
```
before issueing the command that causes the error?
Could you please share the output of 
```
BiocManager::version()
sessionInfo()
```

**Comment by akettring** [comment]
Thanks for the fast response. I've been trying to make this work for about a week. Today I installed the github version. Restarted R. Still didn't work. Started a new session. Now it works. I'm going to blame my IT department and close the issue. Sorry for any confusion.
